benchmarks/proto/test.pb.go

```diff
@@ -416,16 +416,16 @@ func (m *ResultStatus) MarshalTo(data []byte) (int, error) {
 	}
 	if m.PubKey == nil {
 		return 0, github_com_gogo_protobuf_proto.NewRequiredNotSetError("pubKey")
-	} else {
```

Comment:
```
// Code generated by protoc-gen-gogo.
// source: test.proto
// DO NOT EDIT!
```

Reply:
Oh, I miss that! I will revert the changes...